packages/fxa-settings/src/pages/Signin/SigninTokenCode/en.ftl

```diff
@@ -15,3 +15,5 @@ signin-token-code-code-expired = Code expired?
 # Link to resend a new code to the user's email.
 signin-token-code-resend-code-link = Email new code.
 signin-token-code-required-error = Confirmation code required
+# Error displayed in a tooltip when the form is submitted without a code
+signin-token-code-required-error = Confirmation code required
```

Comment:
Is this a duplicate ID or is it appearing in a different context from the one on line 17?
`signin-token-code-required-error`

Reply:
Unintentional, thanks for spotting this!